docs/articles/examples.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: akka-examples
+title: Example Projects 
+---
+
+Here we will be showing a curated lists of projects that use various modules of Akka.NET
+
+## Fire Alert System
```

Comment:
Section header should be: "Akka.Streams" and then have a bulleted list of GitHub repositories, like this

```
[Repo name](repo url) - brief description, including which Akka.NET modules are used.
```